src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
@teohhanhui it was called `web-link` and he asked for a change :) https://github.com/symfony/symfony/pull/22273#discussion_r110030764
I've no strong opinion about that even if I prefer with the caret.